BUILD.md

```diff
@@ -48,7 +48,7 @@ At the time of this writing there is no straightforward option to [set the docke
 ```
 curl -s https://raw.githubusercontent.com/openshift/openshift-ansible/master/images/installer/Dockerfile |
      oc new-build -D - \
-        --docker-image=docker.io/aweiteka/playbook2image \
+        --docker-image=registry.centos.org/openshift/playbook2image \
 	    https://github.com/openshift/openshift-ansible
```

Comment:
Good point. But actually, thinking a bit more about it: I think it would be a good idea to use `openshift-ansible` as the image name in the build instructions to reduce confusion with released images, and also add some more information about the potentially confusing new image names...

Going to submit some changes along those lines.